apps/docs-snippets/src/guide/wallets/connectors.test.ts

```diff
@@ -140,17 +140,17 @@ class WalletConnector extends FuelConnector {
  */
 describe('connectors', () => {
   describe('instantiation', () => {
-    test('should be able to instantiate a Fuel SDK', () => {
+    test('should be able to instantiate a Fuel SDK', async () => {
       // #region fuel-instantiation-1
       // #import { Fuel };
 
-      const sdk = new Fuel();
+      const sdk = await new Fuel().init();
```

Comment:
Nit - just trying to make the `init` method more evident in the docs.

```suggestion
      const sdk = await new Fuel();

      /*
        Awaits for initialization to mitigate potential race conditions
        derived from the async nature of instantiating a connector.
      */
      await sdk.init(); 
```

Reply:
Edited/fixed the code suggestion.

Reply:
Good call